UPGRADING.md

```diff
@@ -3,6 +3,30 @@ Upgrading Grape
 
 ### Upgrading to >= 0.14.0
 
+#### Changes to behavior of `after` method of middleware on error
```

Comment:
This should be under >= 0.15.0 and increment the version in version.rb to 0.15.0 as well, now that we have a major change.